src/suggestion.js

```diff
@@ -0,0 +1,42 @@
+
+export class Suggestion {
+    static create(suggestion){
+       return fetch ('https://app-js-by-yk-default-rtdb.europe-west1.firebasedatabase.app/suggestion.json', {
+            method: 'POST',
+            body: JSON.stringify(suggestion),
+            headers: {
+                'Content-Type': 'application/json'
+            }
+        })
+        .then(response => response.json())
+        .then(response => {
+            suggestion.id = response.name
+            return suggestion
+        })
+    }
+
+    static fetch(token, currentUser) {
+        if (!token){
+            return Promise.resolve(console.log('No token'))
```

Comment:
You should throw an error in this case like 
`return Promise.reject(new Error('No Token!'))`